CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ installing all the necessary `opam` dependencies.
 The target `make watch` will start the Dune build watcher, and recompile the
 project as you save files.
 
-You can run tests with `make test`, and reformat the source code with `make
+You can run tests with `make install test`, and reformat the source code with `make
```

Comment:
I included this change because at least on my machine, I need to run `install` to have tests pass. Maybe it's a misconfiguration issue just on my machine though?